function/spring-computer-vision-function/README.adoc

```diff
@@ -0,0 +1,214 @@
+= Computer Vision Functions
+
+This module provides functional interface to perform common Computer Vision tasks such as Image Classification, Object Detection, Instance and Semantic Segmentation, Pose Estimation an more.
+
+It leverages the https://docs.djl.ai/index.html[Deep Java Library] (DJL) library to enable Java developers to harness the power of deep learning.
+DJL serves as a bridge between the rich ecosystem of Java programming and the cutting-edge capabilities of deep learning.
+DJL provides integration with popular deep learning frameworks like `TensorFlow`, `PyTorch`, and `MXNet`, as well as support for a variety of pre-trained models using `ONNX Runtime`.
+
+== Beans for injection
+
+This module exposes auto-configurations for the following beans:
+
+* `Function<Message<byte[]>, Message<byte[]>> objectDetection` - Offering `Object Detection` for finding all instances of objects from a known set of categories in an image and `Instance Segmentation`` for finding all instances of objects from a known set of categories in an image and drawing a mask on each instance.
```

Comment:
Extra back-tick in the `Instance Segmentation` code snippet.
Although we need to revise if all those words indeed have to present as a code, rather than *bold* or _italic_